slashing-protection/src/testFixtures/java/db/DatabaseUtil.java

```diff
@@ -26,21 +27,26 @@
   public static final String PASSWORD = "postgres";
 
   public static TestDatabaseInfo create() {
+    final TestDatabaseInfo testDatabaseInfo = createWithoutMigration();
+    final Flyway flyway =
+        Flyway.configure()
+            .locations("/migrations/postgresql/")
```

Comment:
may be we can extract `/migrations/postgresql/` as a constant? It seems it is used almost everywhere.

Reply:
done